tests/lib/dirs.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
 # Default applies for: Ubuntu, Debian
-SNAPMOUNTDIR=/snap
-LIBEXECDIR=/usr/lib
+export SNAPMOUNTDIR=/snap
```

Comment:
because shellcheck couldn't tell if this was meant to be sourced or run, and neither could I :-)